Migration.lua

```diff
@@ -128,6 +136,21 @@ local function UpdatePoints(uid, targets, value)
     entry:setTime(timestamp)
     entry:setCounter(timestampCounter[timestamp])
     LedgerManager:Submit(entry)
+
+    if (spent ~= nil) then
```

Comment:
that's doubling the code a lot. I'd rather see the whole `UpdatePoints` called twice in the migration method, with new parameter `UpdatePoints(uid, targets, value, isSpent)` that will propagate to the ledger entry creation.

Reply:
Fair enough, will update